Dockerfile

```diff
@@ -3,17 +3,13 @@ FROM python:3.7-slim-buster
 
 # Build options
 ARG hive_version=2.3.7
-ARG spark_version=3.0.0
+ARG spark_version=3.1.2
```

Comment:
spark 3.1.x has an issue with glue (date parition) https://github.com/awslabs/aws-glue-data-catalog-client-for-apache-hive-metastore/issues/45
3.0.2 is ok